src/transports/ipc/bipc.c

```diff
@@ -168,6 +168,11 @@ static void nn_bipc_destroy (struct nn_epbase *self)
 static void nn_bipc_shutdown (struct nn_fsm *self, int src, int type,
     void *srcptr)
 {
+#if !defined NN_HAVE_WINDOWS
```

Comment:
It would be better to see not not-logic here -- and also, ideally a symbol that represents a feature, not a platform. Based on our current symbol definitions from CMake, would this be reasonable? Do we need more symbols? Or even another CMake option such as `NN_ENABLE_RELEASE_PIPE`  (that defaults to `ON`) ?